css-brochure-form/styles.css

```diff
@@ -0,0 +1,49 @@
+.lmore{
+  color:rgb(255, 123, 0);
+  padding-left: 110px;
```

Comment:
Are you using this to center the header? Try using the `text-align` property instead.